tests/misc/tstripinplace.nim

```diff
@@ -0,0 +1,38 @@
+discard """
+  matrix: "c js"
```

Comment:
that can't work; I wonder how can CI be green?
s/matrix/targets/

Reply:
"matrix" exists too.

Reply:
I know, that's not my point; matrix is used for flags, not commands. This cannot work, it'll end up with a cmdline invocation like `c js`